### PR TITLE
Experiment: use `impl Into<Local<'a, T>>`

### PR DESCRIPTION
**Pull request by piscisaureus**

### CODE REVIEW

**Comment by bartlomieju** [comment]
This is very ergonomic :+1:

**Comment by piscisaureus** [comment]
> This is very ergonomic 👍

Thanks :D I like it too but there are some rough edges. I think I'll finish it when the more important things are done.

File: tests/test_api.rs [hunk 1]
```diff
@@ -1692,9 +1678,9 @@ fn dynamic_import() {
     let mut cbs = v8::CallbackScope::new(context);
     let mut hs = v8::HandleScope::new(cbs.enter());
     let scope = hs.enter();
-    assert!(specifier.strict_equals(v8_str(scope, "bar.js").into()));
+    assert!(specifier.strict_equals(v8_str(scope, "bar.js")));
     let e = v8_str(scope, "boom");
-    scope.isolate().throw_exception(e.into());
+    scope.isolate().throw_exception(e);
```
Comment (ry) [inline tests/test_api.rs]: very nice!

**Comment by piscisaureus** [comment]
Closing for now.

I still want to do this some day, but we'll add it piecemeal, and probably after defining a specific trait for it (IntoLocal) rather than using the generic Into trait.

**Pull request closed without merging**